src/constants/gatsby-utils.js

```diff
@@ -265,6 +274,20 @@ const configureRedirects = (toPath, redirects, actions, config = {}) => {
         config,
       ),
     );
+    if (toIsLatest) {
+      actions.createRedirect(
+        Object.assign(
+          {
+            fromPath: replacePathVersion(fromPath),
+            toPath: replacePathVersion(toPath),
+            redirectInBrowser: true,
+            isPermanent: false,
+            force: true,
+          },
+          config,
```

Comment:
not a required change, maybe this could be done using ES6 syntax?

```javascript
actions.createRedirect({
              ...config,
              fromPath: replacePathVersion(fromPath),
              toPath: replacePathVersion(toPath),
              redirectInBrowser: true,
              isPermanent: false,
              force: true,
})
```

Reply:
That reverses the behavior! Right now, "config" can be used to *override* whatever properties are set in this function - but this could still be done using spread syntax, e.g. 

```javascript
{
  fromPath: replacePathVersion(fromPath),
  toPath: replacePathVersion(toPath),
  redirectInBrowser: true,
  isPermanent: false,
  force: true,
  ...config,
}
```

...of course, this functionality is never used so far as I can tell - this method is only called once, and no config arg is provided. I have... No idea what the intent was TBH. I might be better off dropping it entirely; would simplify the code.

Reply:
👌